### PR TITLE
Embedding Images and Video

### PR DESCRIPTION
**Pull request by realisticattorney**
Hello. We made an initial mistake of creating a branch “Development” from the local main. From that point on, most commitings had no point to comparison to be visible on the pull request. However, you can see who commited what from this link. 
https://github.com/realisticattorney/YoutubePageRepo/commits/Development

Me and my partner tried to solve it with the help of my mentor and his coding partner with no success, for about two hours.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,285 @@
+*{
+   box-sizing: border-box;
+   cursor: pointer;
+   font-family: 'Roboto', sans-serif;
+   
+}
+body {
+   font-family: 'Montserrat', sans-serif;
```
Comment (ioilmio) [inline styles.css:8]: Which font family are using?
Mont-serrat or Roboto?
Choose one in this case or add a specific font family to a specific element;

**Review by ioilmio — CHANGES_REQUESTED** [review]
# 👋🏼  Hello @realisticattorney and @alexisbec 
## Good work! you have put the right amount of effort for your first project together 💯 
### You are using semantic HTML tag correctly and your project looks good, but there are some point you need to fix;

# Here they are 📄 :

Your Github flow is partially wrong i suggest to take a look at [this](https://www.theodinproject.com/courses/web-development-101/lessons/introduction-to-git)

There are some issue to fix and it will be ok 👌🏼  , so please follow the comments I left for you below this window, in your code and fix the following list of requirements
## Changes requested :  
- [ ] Please make sure your project has a descriptive README.md file that explain and showcase your project with useful information, a good example can be found at [this link](https://github.com/microverseinc/readme-template)
- [ ] Pull Request has a meaningful title and a brief description, be more specific about what your PR is for;
- [ ] You commits messages can be more descriptive;
For the above two points, I suggest you review your onboarding material and the very first two section of the technical curriculum;

- [ ] The video is displayed but is not available, i can't view it;
 Please try with a different one;

- [ ] In your icon folder there are images that are not actually used in the project, please remove them

## Optional:

- [ ] Your project layout in a browser that is not Chrome is broken;
I used firefox to open it the first time and it was pretty different;

- [ ] You can level up you CSS game, when you are in doubt I suggest you [this guides](https://css-tricks.com/guides/) to start;

For any question you can contact me;
# Thank you ✊🏼 
# Happy coding 💻

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
### Hello team :wave: 
Great work you guys have done in implementing the changes requested by the previous TSE. Your project looks good :clap: 
#### However there is still one final change to be made to get the project approved.
- [ ] You need to have a descriptive pull request title and description. Right now your pull request description is not talking about the project itself. Update your pull request, you can give a brief description of the project you've just built.
- [ ] [Optional] You can include a live link to your project. You can deploy your project to GitHub pages or use http://raw.githack.com/ to make it accessible online.

### Please request another review as soon as you've done the required changes.
## Happy Coding :+1: 
slack(@ebeagusamuel)

**Comment by realisticattorney** [comment]
Project Milestones:

- Giflow;

- Implementation of proper semantic elements

- Used Flex for positioning

- Embeded video player

- Added LICENSE.

- Descriptive Readme.md file

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # YoutubePageRepo
-embedding the YouTube video player into the page so it actually plays and showing little thumbnail images along the right side.
+
+Embedding the YouTube video player into the page so it actually plays and showing little thumbnail images along the right side.
+
+![screenshot](icons/screenshot.jpg)
```
Comment (descholar-ceo) [inline README.md:8]: This screenshot is not visible, please make sure that you add an accessible screenshot here, which is in your project's folder

File: README.md [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+It shows the video title.
+It shows the number of views.
+It has like and dislike icons.
+It has a share button.
+It shows the avatar of the user who uploaded the video.
+It has a subscribe button.
+Sidebar Recommended videos
+It shows the recommended video’s thumbnail preview (as image not as an actual video).
+It shows the title of the video.
+It shows the number of views.
```
Comment (descholar-ceo) [inline README.md:39]: This description is good, but it seems to not meet good project description, a non-tech user won't be able to understand, and this is the complete copy paste from your pathwright project specifications, that is not allowed. You should write project specification from your own words.

Actually that is not difficult, just try to think about what you have done on this project, and explain it to your grand-mother and make sure that she has understaood what you meant, that's what is a project description. Try to exercise yourself while describing a project in that way

File: README.md [hunk 4]
```diff
@@ -1,2 +1,74 @@
...
+## Built With
+
+HTML & CSS
+
+
+## Authors
+
+👤 German Axel Aquila
+
+- GitHub: [@realisticattorney](https://github.com/realisticattorney)
```
Comment (descholar-ceo) [inline README.md:50]: If you have some other social links such as `twitter` or `linkedin`, kindly add them here as well

File: README.md [hunk 5]
```diff
@@ -1,2 +1,74 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+To Microverse, the stand-up team, and mentor
+
+## 📝 License
+
+This project is [MIT](http://angularjs.org) licensed.
```
Comment (descholar-ceo) [inline README.md:74]: This is not a valid lincense link, please fix this to point it to the `The MIT License.txt` file you have in root directory of your project

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @realisticattorney and @alexisbec  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]

- [x] Don't use ID selectors in your CSS
- [ ] As it is suggested in the previous reviews, please work on your readme file on the `description section`
- [ ] Please make sure that the screenshot on your readme file is visible
- [ ] Fix the license link in readme file
- [ ] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [ ] It will be nice if `inputs`, `buttons`, and all of other form elements are implemented inside a `form element`
- [ ] Your commit messages should be in imperative mode or present tense, you should have used `Finish with first breakpoint(767px)` instead of `finishing with first breakpoint(767px)`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: README.md [hunk 6]
```diff
@@ -1,2 +1,51 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # YoutubePageRepo
-embedding the YouTube video player into the page so it actually plays and showing little thumbnail images along the right side.
+
+Embedding the YouTube video player into the page so it actually plays and showing little thumbnail images along the right side. 
+
+https://github.com/realisticattorney/YoutubePageRepo/blob/feature/icons/screenshot.png
```
Comment (descholar-ceo) [inline README.md:8]: This screenshot is still not showing up on your readme file, [refer to this article](https://medium.com/markdown-monster-blog/getting-images-into-markdown-documents-and-weblog-posts-with-markdown-monster-9ec6f353d8ec) On how to display images in mark down language

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @realisticattorney and @alexisbec  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [ ] Please fix the screenshot which is not displayed from your readme file
[refer to this article](https://medium.com/markdown-monster-blog/getting-images-into-markdown-documents-and-weblog-posts-with-markdown-monster-9ec6f353d8ec)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by jubaan — APPROVED** [review]
# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

#### :clap: :clap: Great job :hammer_and_pick:!! There are no more issues left to be addressed.  

## You may now merge to master! :+1: :100:

## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

**Pull request merged**